engine/src/systems/lights/SpotLightsSystem.cpp

```
@@ -18,23 +18,31 @@
 #include "components/SceneComponents.hpp"
 #include "ecs/Coordinator.hpp"
 
-
 namespace nexo::system {
-	void SpotLightsSystem::update() const
+	void SpotLightsSystem::update()
```

Cppcheck (2.10-2)

[style] The function 'update' is never used.

(unusedFunction)